src/execution_tree/primitives/annotate_primitive.cpp

```diff
@@ -118,25 +119,23 @@ namespace phylanx { namespace execution_tree { namespace primitives {
             {
                 HPX_THROW_EXCEPTION(hpx::bad_parameter,
                     "annotate_primitive::annotate_d",
-                    generate_error_message("locality and/or tile annotation not found"));
+                    generate_error_message(
+                        "locality and/or tile annotation not found"));
             }
         }
+        else if (phylanx::execution_tree::extract_string_value(*args.begin()) ==
+            "tile")
+        {
```

Comment:
Is `the first element in `args` guaranteed to refer to the `tile` annotation?

Reply:
This is in the case where there is no args wrapper, and only the tile annotation is supplied.